tools/benchmark/package.json

```diff
@@ -31,7 +35,7 @@
     "benchmark": "^2.1.4",
     "chai": "^4.2.0",
     "easy-table": "^1.1.1",
-    "mocha": "^8.4.0"
+    "mocha": "^8.1.1"
```

Comment:
@CraigMacomber Asking you since you touched this project last... the mocha dependency was duplicated in deps and devDeps, which I assume was unintentional. Do you know if there was a reason for the duplicated dependency?